website/docs/r/compute_network.html.markdown

```diff
@@ -17,6 +17,16 @@ resource "google_compute_network" "default" {
   name                    = "foobar"
   auto_create_subnetworks = "true"
 }
+
+resource "google_compute_network" "other" {
```

Comment:
Same as with the tests - let's make this example a valid network peering.

If we can have one network automatically create subnetworks and we manually define them for another & manage to peer successfully, we'd be able to show even more of how to use this resource to readers.